modules/openapi-generator/src/main/resources/r/api_client.mustache

```diff
@@ -69,22 +71,32 @@ ApiClient  <- R6::R6Class(
       if (!is.null(userAgent)) {
         self$`userAgent` <- userAgent
       }
+
+      if (!is.null(timeout)) {
+        self$timeout <- timeout
+      }
     },
     CallApi = function(url, method, queryParams, headerParams, body, ...){
       headers <- httr::add_headers(c(headerParams, self$defaultHeaders))
 
+      {{! Adding timeout that can be set at the apiClient object level}}
+      httrtimeout <- NULL
```

Comment:
Minor suggestion: use `httpTimeout` instead to follow other parameters' naming convention, use `http` instead so as to avoid the name tied to the HTTP library `httr`

Reply:
fixed it